CoreFoundation/String.subproj/CFString.h

```diff
@@ -174,10 +174,17 @@ struct __CFConstStr {
 #define CONST_STRING_LITERAL_SECTION
 #endif
 
+#if __BIG_ENDIAN__
+#define CFSTR(cStr)  ({ \
+static struct __CFConstStr str CONST_STRING_LITERAL_SECTION = {{(uintptr_t)&__CFConstantStringClassReference, _CF_CONSTANT_OBJECT_STRONG_RC, 0, {0x00, 0x00, 0x07, 0xc8}, {0x00, 0x00, 0x00, 0x00}}, (uint8_t *)(cStr), sizeof(cStr) - 1}; \
+(CFStringRef)&str; \
+})
```

Comment:
This is not the only macro btw, there is another in CFInternal.h called `CONST_STRING_DECL` that probably should be double checked if it is still correct; it has been a while since CF ran on big endian.